nym-wallet/src-tauri/src/wallet_storage/mod.rs

```diff
@@ -152,6 +153,48 @@ pub(crate) fn store_login_with_multiple_accounts(
     store_login_with_multiple_accounts_at_file(&filepath, mnemonic, hd_path, id, password)
 }
 
+/// Update all logins with multiple accounts support
+pub(crate) fn update_encrypted_logins(
+    current_password: &UserPassword,
+    new_password: &UserPassword,
+) -> Result<(), BackendError> {
+    // make sure the entire directory structure exists
+    let store_dir = get_storage_directory()?;
+    create_dir_all(&store_dir)?;
+    let filepath = store_dir.join(WALLET_INFO_FILENAME);
```

Comment:
Just check if it exists? If you try to update something that doesn't exists then that seems like an error?